rust/decentraland-godot-lib/src/dcl/js/js_modules/RestrictedActions.js

```diff
@@ -1,7 +1,9 @@
 module.exports.movePlayerTo = async function (body) { return {} }
 module.exports.teleportTo = async function (body) { return {} }
 module.exports.triggerEmote = async function (body) { return {} }
-module.exports.changeRealm = async function (body) { return {} }
+module.exports.changeRealm = function (body) {
+    return Deno.core.ops.op_change_realm(body.realm, body.message);
+}
```

Comment:
```suggestion
module.exports.changeRealm = async function (body) {
    const response = await Deno.core.ops.op_change_realm(body.realm, body.message)
    return response
}
```

It's not bad as you put, but I'd write more verbosely because as we're in JavaScript and not TypeScript.